docs/_documentations/remotedeploy-single.md

```diff
@@ -115,16 +113,16 @@ A complete command will look something like this :
 --kurl https://codewind-keycloak-k55dxqhx.mycluster-12345-7674b4bd9abbdeea5be228236d5275c9-0001.eu-gb.containers.appdomain.cloud
 ```
 
-Which will:
+Which:
```

Comment:
If we translate this file into another language in the future, I'm thinking "which" might be confusing. A complete sentence like "This command performs the following actions:" might be clearer...